config/routes.rb

```diff
@@ -119,19 +119,15 @@
         end
       end
       resources :follows, only: [:create]
-      resources :followers do
-        collection do
-          get :users
-          get :organizations
-        end
+      namespace :followers do
+        get :users
+        get :organizations
       end
-      resources :followings do
-        collection do
-          get :users
-          get :tags
-          get :organizations
-          get :podcasts
-        end
+      namespace :followings do
+        get :users
+        get :tags
+        get :organizations
+        get :podcasts
```

Comment:
this have been changed into namespaces because we are using none of the CRUD operations created by `resources`. We don't need to generate all those routes